src/main/kotlin/org/jitsi/nlj/util/PacketInfoQueue.kt

```diff
@@ -24,7 +24,7 @@ class PacketInfoQueue(
     id: String,
     executor: ExecutorService,
     handler: (PacketInfo) -> Boolean
-) : PacketQueue<PacketInfo>(100, false, false, id, handler, executor) {
+) : PacketQueue<PacketInfo>(100, false, null, id, handler, executor) {
```

Comment:
what is this change?

Reply:
In order to enable/disable the stats globally I added an overridable default. I changed the parameter type to Boolean and now passing `null` makes it use the default (which the bridge updates): https://github.com/jitsi/jitsi-utils/commit/9c4e610c5a0623c62e83282e770683503113f4c4

Reply:
I think we might want to refactor that in the future, and having another constructor would make it harder.